server/README.md

```diff
@@ -1,6 +1,63 @@
 # INS Server
 The INS server provides a backend RESTful service for INS-node and front-end web or mobile applications.
 
+## API
+* Send RSSI value to INS server (INS node)
+
+  INS node is required to send RSSI reading per wifi AP's mac address to INS service. In a bid to handle outliers, the server supports sending multiple data points from INS node.
+
+  HTTP Method - `POST`
+
+  Request Url -  `/set_rss/:device_id/:ssid1/:rssi1/:ssid2/:rssi2/:ssid3/:rssi3/:ssid4/:rssi4/:end?`
+
+  Response - `{result:success}` or `{result:error}`
+
+  #### Examples
+  * Device `1000` sending a one-time RSSI reading from SSIDs `aa`, `bb`, `cc` and `dd`.
```

Comment:
But, are "`aa`, `bb`, `cc` and `dd`" really SSID's? Aren't they AP MAC addresses?

Reply:
Hmm... They are in fact MAC address since there will be same SSIDs for several APs. I have to do some rename in the code to reflect this as well. Good point!